devices/Bq2579x/Bq2579x.cs

```diff
@@ -0,0 +1,939 @@
+        /// </para>
+        /// <para>
+        /// Range: 3600mV-22000mV.
+        /// </para>
+        /// </remarks>
+        public ElectricPotentialDc InputVoltageLimit { get => GetInputVoltageLimit(); set => SetInputVoltageLimit(value); }
+
+        /// <summary>
+        /// Gets or sets Input Current Limit.
+        /// </summary>
+        /// <exception cref="ArgumentOutOfRangeException">If value is out of range (100mA-3300mA).</exception>
+        /// <remarks>
+        /// <para>
+        /// Based on D+/D- detection results:
+        /// USB SDP = 500mA
+        /// USB CDP = 1.5A
+        /// USB DCP = 3.25A
+        /// Adjustable High Voltage DCP = 1.5A
+        /// Unknown Adapter = 3A
+        /// Non-Standard Adapter = 1A/2A/2.1A/2.4A
+        /// </para>
+        /// <para>
+        /// Range: 100mA-3300mA.
+        /// </para>
+        /// </remarks>
+        public ElectricCurrent InputCurrentLimit { get => GetInputCurrentLimit(); set => SetInputCurrentLimit(value); }
+
+        /// <summary>
+        /// Gets or sets battery voltage thresholds for the transition from precharge to fast charge.
+        /// </summary>
+        /// <remarks>Defined as a ratio of battery regulation limit(VREG).</remarks>
+        [Property]
+        public ThresholdFastCharge FastChargeTransitionVoltage { get => GetThresholdFastCharge(); set => SetThresholdFastCharge(value); }
+
+        /// <summary>
+        /// Gets or sets precharge current limit (in steps of 40mA).
+        /// </summary>
+        /// <exception cref="ArgumentOutOfRangeException">If value is out of range (40mA-2000mA).</exception>
+        /// <remarks>Equivalent range: 40mA-2000mA.</remarks>
+        [Property]
+        public ElectricCurrent PrechargeCurrentLimit { get => GetPrechargeCurrentLimit(); set => SetPrechargeCurrentLimit(value); }
+
+        /// <summary>
+        /// Gets VBUS.
+        /// </summary>
+        /// <remarks>Range: 0mV-30000mV.</remarks>
+        public ElectricPotentialDc Vbus => GetAdcVoltage(Register.REG35_VBUS_ADC);
+
+        /// <summary>
+        /// Gets VAC1.
+        /// </summary>
+        /// <remarks>Range: 0mV-30000mV.</remarks>
+        public ElectricPotentialDc Vac1 => GetAdcVoltage(Register.REG37_VAC1_ADC);
+
+        /// <summary>
+        /// Gets VAC2.
+        /// </summary>
+        /// <remarks>Range: 0mV-30000mV.</remarks>
+        public ElectricPotentialDc Vac2 => GetAdcVoltage(Register.REG39_VAC2_ADC);
+
+        /// <summary>
+        /// Gets VBAT.
+        /// </summary>
+        /// <remarks>Range: 0mV-20000mV.</remarks>
+        public ElectricPotentialDc Vbat => GetAdcVoltage(Register.REG3B_VBAT_ADC);
+
+        /// <summary>
+        /// Gets VSYS.
+        /// </summary>
+        /// <remarks>Range: 0mV-24000mV.</remarks>
+        public ElectricPotentialDc Vsys => GetAdcVoltage(Register.REG3D_VSYS_ADC);
+
+        /// <summary>
+        /// Gets die temperature.
```

Comment:
what die stands for?

Reply:
Die is not an abreviation is the name given to the chip core.